redis-om-spring/src/main/java/com/redis/om/spring/convert/BooleanToBytesConverter.java

```diff
@@ -0,0 +1,23 @@
+package com.redis.om.spring.convert;
```

Comment:
If this level of conversation is problematic, to the point that we have to write multiple conversions - IMHO they all belong in Jedis itself.  This is generally the pattern relied upon in the python ecosystem.

Reply:
No exactly, these are Spring Data-specific converters based on the needs for searching. Spring Data provides a pluggable converter API which Spring Data Redis extends, and we further extend it to ensure the conversions for Hashes match the conversions we do for JSON documents. Therefore ensuring consistent serialization/deserialization between Hashes and JSON mapped entities.

Reply:
See https://github.com/spring-projects/spring-data-redis/blob/main/src/main/java/org/springframework/data/redis/core/convert/BinaryConverters.java